src/components/AddWalletModal/AddWalletModal.jsx

```diff
@@ -18,6 +18,7 @@ import {
   Text,
   useDisclosure
 } from '@chakra-ui/react';
+import axios from 'axios';
```

Comment:
we have an api folder. Put youre request in there and wrap your query in a useQuery hook